insolar/pulse/start_pulse.go

```diff
@@ -0,0 +1,34 @@
+package pulse
+
+import (
+	"context"
+
+	"github.com/insolar/insolar/insolar"
+	"errors"
+)
+
+type StartPulse interface {
+	OnPulse(context.Context, insolar.Pulse)
```

Comment:
`OnPulse` means "do whatever you want". It's not very clear.
Let's make it explicit `SetStartPulse(context.Context, insolar.Pulse)`

Reply:
done